pkg/apis/core/types.go

```diff
@@ -2891,6 +2907,16 @@ type PodSecurityContext struct {
 	// sysctls (by the container runtime) might fail to launch.
 	// +optional
 	Sysctls []Sysctl
```

Comment:
nit: this is in a different place in the versioned API. Can you swap it with FSGroupChangePolicy here?

Reply:
Reordered.